sskittenapi/catapi/tests/domains/test_cat_domain.py

```diff
@@ -144,11 +156,12 @@ async def test_delete_cat_error_entity_not_found(mock_cat_model_delete_cat: mock
 @conftest.async_test
 async def test_partial_update_cat(
     mock_cat_model_partial_update_cat: mock.Mock,
+    expected_cat: dto.Cat,
     partial_update_cat: dto.PartialUpdateCat,
     cat_id: dto.CatID,
 ) -> None:
-    excepted_cat = dto.Cat
-    mock_cat_model_partial_update_cat.return_value = excepted_cat
+    expected_cat = expected_cat
```

Comment:
I think this line is redundant and can be removed.
WDYT?

Reply:
_A quick check_: When I remove line # 151 Then it fails the UT.